Parsers/ASimNetworkSession/Tests/SonicWall_SonicOS_ASimNetworkSession_DataTest.csv

```diff
@@ -0,0 +1,29 @@
+﻿Result
+"(0) Error: 1 invalid value(s) (up to 10 listed) in 1000 records (100.0%) for field [DvcIpAddr] of type [IP Address]: [""18C24135E8B6""] (Schema:NetworkSession)"
+"(0) Error: 1 invalid value(s) (up to 10 listed) in 1000 records (100.0%) for field [EventProduct] of type [Enumerated]: [""SonicOS""] (Schema:NetworkSession)"
+"(0) Error: 1 invalid value(s) (up to 10 listed) in 931 records (93.1%) for field [DvcAction] of type [Enumerated]: [""NA""] (Schema:NetworkSession)"

```

Comment:
please keep 'Other' if not matching with allowed values

Reply:
EventProduct is correct and should be accepted.

DvcAction was addressed by using "Other" as requested in another comment.

DvcIpAddr has been removed. I've observed it sometimes sent as an IP and other times as a different identifier. To err on the safe side. I've removed the field.